exchange/index.go

```diff
@@ -453,6 +477,88 @@ func (idx *Index) evict(size uint64) uint64 {
 	return evicted
 }
 
+// tagForEviction tags CIDs that will be evicted during garbage collection
+func (idx *Index) tagForEviction(ref *DataRef) error {
```

Comment:
```suggestion
func (idx *Index) tagForGC(ref *DataRef) error {
```
I think it would be less confusing to keep the word `eviction` as in evicting from the LFU and use GC since it is for garbage collection and this tagged content has already been evicted.